### PR TITLE
added descriptions for the options

### PR DESCRIPTION
**Pull request by owolabileg**

### CODE REVIEW

File: nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/AbstractNondexMojo.java [hunk 1]
```diff
@@ -58,37 +58,83 @@ a copy of this software and associated documentation files (the
...
 public abstract class AbstractNondexMojo extends AbstractMojo {
     // NonDex Mojo specific properties
+    /**
+     * The seed that is used for randomization during shuffling.
+     */
     @Parameter(property = ConfigurationDefaults.PROPERTY_SEED, defaultValue = ConfigurationDefaults.DEFAULT_SEED_STR)
     protected int seed;
 
+    /**
+     * The degree/level of shuffling that should be carried out.
```
Comment (alexgyori) [inline nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/AbstractNondexMojo.java]: This we will be shrank to only FULL and ONE soon.

File: nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/AbstractNondexMojo.java [hunk 2]
```diff
@@ -58,37 +58,83 @@ a copy of this software and associated documentation files (the
...
     protected boolean rerun;
 
+    /**
+     * Unique ID for the current nondex execution.
+     */
     @Parameter(property = ConfigurationDefaults.PROPERTY_EXECUTION_ID,
             defaultValue = ConfigurationDefaults.PROPERTY_DEFAULT_EXECUTION_ID)
     protected String executionId;
 
+    /**
```
Comment (alexgyori) [inline nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/AbstractNondexMojo.java]: Which run you want to debug for. A run is composed of a number of executions. You specify which run you want to debug and the debug phase will take all the failing tests across executions in that run and debug them. It should be able to debug a single executions, not only a set of executions.

**Comment by alexgyori** [comment]
It looks fine, just replace the TODO with a meaningful description.

**Comment by owolabileg** [comment]
Thanks, @alexgyori. That TODO is assigned to Gyori. Let's hope he addresses it soon :) Alternatively you could merge this and let him make a separate pull request.

**Comment by alexgyori** [comment]
I fixed the todo @owolabileg.

**Pull request merged**